packages/server/api/controllers/movies.controller.js

```diff
@@ -25,6 +25,22 @@ const getMovieByID = async (id) => {
   }
 };
 
+const getReviewsByMovieID = async (id) => {
+  if (!id) {
+    throw new HttpError('ID should be a number', 400);
+  }
+
+  try {
+    const reviews = await knex('reviews').select('*').where({ movie_id: id });
```

Comment:
You could simply this select by just removing select('*').
Refer cheatsheet https://devhints.io/knex

Reply:
Thanks for the feedback and sorry for the long delay with the answer. We have agreed in our team to change a logic and tasks allocation. So this PR is now duplicated by another one. I have to close it (but will keep your comments noted for further code).